### PR TITLE
Feature/sc 9189 categories task dashboard teacher

### PR DESCRIPTION
**Pull request by dominikbruechner**
# Short Description
Als LehrerIn möchte ich eine eine Liste meiner Aufgaben sehen, die in verschiedene Kategorien (ohne/mit Abgabefrist) unterteilt ist.

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/organisms/HomeworksDashboardTeacher.vue [hunk 1]
```diff
@@ -1,20 +1,58 @@
...
-			:homeworks="dueDateHomeworks"
-			:title="$t('pages.homeworks.teacher.subtitleAssigned')"
-			type="teacher"
-		/>
-		<homeworks-list
-			:homeworks="noDueDateHomeworks"
-			:title="$t('pages.homeworks.teacher.subtitleNoDue')"
-			type="teacher"
-		/>
+		<v-expansion-panels v-model="expanded" flat accordion mandatory multiple>
```
Comment (pkleybolte) [inline src/components/organisms/HomeworksDashboardTeacher.vue:3]: If you used classes for the expansion panels, you could distinguish them in the test

File: src/components/organisms/HomeworksDashboardTeacher.unit.js [hunk 2]
```diff
@@ -37,6 +62,53 @@ describe("@components/organisms/HomeworksDashboardTeacher", () => {
...
+		const expansionPanels = wrapper.findAll(".v-expansion-panel");
+
+		expect(expansionPanels.exists()).toBe(true);
+		expect(expansionPanels).toHaveLength(1);
+		expect(expansionPanels.at(0).classes()).toContain(
+			"v-expansion-panel--active"
+		);
+	});
+
+	it("Should render only active 'with due date' panel, if the other panel is empty", async () => {
```
Comment (pkleybolte) [inline src/components/organisms/HomeworksDashboardTeacher.unit.js:96]: The expects are basically the same as the last test. Although you are testing something different.

**Review by pkleybolte — APPROVED** [review]

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

**Pull request merged**